cicd/app/email_lambda.yaml

```diff
@@ -0,0 +1,123 @@
+Description: Lambda to send email notification of a token request
+
+Parameters:
+  pArtifactsBucket:
+    Type: String
+    Description: The bucket where the lambda code package is stored
+  pBuild:
+    Type: Number
+    Description: The build number
+    Default: '1'
+  pDestEmail:
+    Type: String
+    Description: The destination email address
+  pCleanBranch:
+    Type: String
+    Description: A cleaned version of the code branch name
+    Default: development
+  pEnvironment:
+    Type: String
+    Description: The AWS environment we're launching into
+    Default: development
+    AllowedValues:
+      - development
+      - testing
+      - staging
+      - production
+  pLambdaCaller:
+    Type: String
+    Description: The ARN of the resource that invokes the lambda
+  pLambdaDeployPackage:
+    Type: String
+    Description: The S3 key of the lambda code package
+  pSourceEmail:
+    Type: String
+    Description: The source email address, must be a pre verified SES address
+
+Outputs:
```

Comment:
How does API gateway know the ARN of the lambda to call? we could export this value so it can pull it in in an automated way

Reply:
For the initial setup, I'd just added the lambda ARN manually to the API Gateway yaml file for the tokens app. 

Yep. I think we will need to export the lamba ARN and somehow and either update the  tokens api's yaml spec file in it's [repo](https://github.com/AtlasOfLivingAustralia/aws-apig-cognito/tree/test/tokens) or update the relevant api gateway itself directly.

Reply:
ok cool, adding it to the API GW yaml file works for now, we'd just have to be sure to update when it's merged that so it's pointing to the correct Lambda environment/ARN.  If we export that the lambda ARN we'd be able to work it into the APIG CICD process. probably easier to make a call to update the API, would have to look into that.

If that API GW is just servicing the tokens app would it make sense for the deploy and config for that to live in the tokens repo? It would simplify the sharing of the Lambda ARN, it would be good to have everything tokens related in the same place too

Reply:
> ok cool, adding it to the API GW yaml file works for now, we'd just have to be sure to update when it's merged that so it's pointing to the correct Lambda environment/ARN. If we export that the lambda ARN we'd be able to work it into the APIG CICD process. probably easier to make a call to update the API, would have to look into that.
> 
> If that API GW is just servicing the tokens app would it make sense for the deploy and config for that to live in the tokens repo? It would simplify the sharing of the Lambda ARN, it would be good to have everything tokens related in the same place too
At this point the API GW content for tokens app is small enough for it to be included in the tokens app repo itself so I dont see any issues with  including it in the tokens app repo.

Reply:
OK great. happy to discuss this too. There's a few things in the air, I'll create a ticket for this